config_utilities_ros/setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
+package_name = "config_utilities_ros"
```

Comment:
Can also drop, this is covered by the use of `ament_cmake_python`. `setup.cfg` can get parsed for entry points and stuff by `ament_cmake_python`, but the support is kinda dicey and requires looking at both the `ament_cmake_python` and `colcon` implementations